src/operator/tensor/elemwise_binary_op.h

```diff
@@ -155,6 +78,15 @@ class ElemwiseBinaryOp : public OpBase {
   };
 
  private:
+  /*!
+   * \brief CSR operation requires temp space
+   */
+  struct ResourceRequest {
```

Comment:
??

Reply:
Used to index into ctx.resources during CSR pass. It is, in fact, referencing a RespurceRequest.